WordPress/WordPress.xcodeproj/project.pbxproj

```diff
@@ -3295,6 +3295,20 @@
 /* End PBXFileReference section */
 
 /* Begin PBXFileSystemSynchronizedBuildFileExceptionSet section */
+		0C62E2312D80C08D009F7E28 /* Exceptions for "Targets" folder in "WordPress" target */ = {
+			isa = PBXFileSystemSynchronizedBuildFileExceptionSet;
+			membershipExceptions = (
+				"Jetpack/Sources/AppDelegate+Jetpack.swift",
```

Comment:
I'd expect the `+WordPress` here

Reply:
I don't know what these are, tbh. `PBXFileSystemSynchronizedBuildFileExceptionSet` ??

Reply:
However... the target membership is correct for the two files, so maybe I'm not understanding this bit of the project correctly

![image](https://github.com/user-attachments/assets/bb3c62c9-2399-439f-b329-b2d2bdbd6a79)
![image](https://github.com/user-attachments/assets/f29f8254-67d9-4f88-834e-359dd437605e)